This patch is needed because gvfs 1.62.0 fails to build on mac, due to
explicit_bzero (clears a password from memory before freeing it), which mac does not have.

Mac has memset_s for the same job, so nothing is lost.

diff --git a/common/gvfsutils.h b/common/gvfsutils.h
index 00097be..bde13ef 100644
--- a/common/gvfsutils.h
+++ b/common/gvfsutils.h
@@ -38,7 +38,11 @@ gvfs_free_password (gpointer str)
 {
   if (str != NULL)
     {
+#ifdef __APPLE__
+      memset_s (str, strlen (str), 0, strlen (str));
+#else
       explicit_bzero (str, strlen (str));
+#endif
       g_free (str);
     }
 }
